lib/schemas.py

```diff
@@ -32,6 +41,14 @@ def parse_message(message_data: Dict) -> Message:
     result_data = body_data.get('result', {})
     if 'yake_keywords' in model_name:
         result_instance = YakeKeywordsResponse(**result_data)
+    elif 'classycat' in model_name:
+        event_type = body_data['parameters']['event_type']
+        if event_type == 'classify':
+            result_instance = ClassyCatBatchClassificationResponse(**result_data)
+        elif event_type == 'schema_lookup' or event_type == 'schema_create':
+            result_instance = ClassyCatSchemaResponse(**result_data)
+        else:
+            result_instance = ClassyCatResponse(**result_data)
     elif 'video' in model_name:
         result_instance = VideoResponse(**result_data)
     else:
```

Comment:
I think this else should be `elif` should raise error if unknown model name  (otherwise will silently fail on typo?)

Reply:
I don't think it will fail silently as is, since we check for the event type when processing the request later. please look at `classycat.py/process()`. 

I did this so I can respond with the same error structure as other classycat request, otherwise you may not receive an error directly from presto or it may be in a different format.

Reply:
oh I was wrong. I understand now this is just mesage validation, I thought it was parsing incoming messages.  I lost a couple of hours debugging because I was sending messages with the model 'yake' instead of 'yake_keywords'  and it just silently accepted ... but this isn't the place to enforce that.

Reply:
hm, for what its worth @skyemeedan we *could* raise an error if the model name is unknown to this function...

Reply:
@DGaffney we do respond with an error to the caller in this case, it's just handled in the process function instead of here. happy to discuss further!

Reply:
Just to clarify, @ashkankzme , I believe Devin was referring to a Presto model (YAKE, Video, Audio, etc.) and I believe you're referring to an LLM model. I think that's a good idea to respond with an error if the Presto model is unknown. That, however, can be a separate ticket / PR